googletest/docs/Pkgconfig.md

```diff
@@ -47,55 +47,6 @@ goes for using `_LDFLAGS` over the more commonplace `_LIBRARIES`, which
 happens to discard `-L` flags and `-pthread`.
 
 
-### Autotools ###
```

Comment:
CMake can (and maybe does?) create pkg-config files. So you *could* (and I think you should) Autotools and Meson by just installing the right `.pc` files.